docs/compute/memory_hotplug.md

```diff
@@ -3,8 +3,12 @@
 Memory hotplug was introduced in KubeVirt version 1.1, enabling the dynamic resizing of the amount of memory available to a running VM.
 
 ## Limitations
-* Memory hotplug is currently only supported on the x86_64 architecture.
+* Memory hotplug is currently only supported on the x86_64,arm64 architectures.
+* Linux guests running at least Linux v5.8 are fully supported.
+* Windows guests support has been added to virtio-win, but it should be considered unstable.
 * Current hotplug implementation involves live-migration of the VM workload.
+* VirtualMachines must have at least 1GiB of memory to support memory-hotplug.
```

Comment:
Is this limitation listed anywhere else (QEMU/Libvirt documentation)? I failed to find any reference.

Reply:
@acardace I see... I didn't review this therefore I wasn't aware of this change. I do wonder about the validating webhook. I'd expect us to simply require a restart via a condition.